src/views/AddPeople.vue

```diff
@@ -2,8 +2,8 @@
   <div class="addblog">
     <div class="container">
       <div class="card card-container">
-        <h1>Enter a new Entry</h1>
-        <h1>{{ this.id }}</h1>
+        <h1 id="header">Enter a new Entry</h1>
+        <h1 id="header">{{ this.id }}</h1>
```

Comment:
Can you remove this please?
This was added as a part of experiment, I forgot to remove later ;-)

Reply:
Remove the ` <h1 id="header">{{ this.id }}</h1>`

Reply:
If you are talking about the "header" id, then they are added by me. I wanted to align the headers at the centers so did that.

Reply:
> Remove the ` <h1 id="header">{{ this.id }}</h1>`

@BAAHUBALI7781  What's the confusion? I wrote it quite clear to remove the above line :confused: